rdmo/projects/validators.py

```diff
@@ -28,13 +28,21 @@ def __call__(self, data, serializer):
                     })
         else:
             # for a new value, check if there is already a value with the same attribute and indexes
+            get_kwargs = {
+                'attribute': data.get('attribute'),
+                'set_prefix': data.get('set_prefix'),
+                'set_index': data.get('set_index'),
+                'collection_index': data.get('collection_index')
+            }
+
+            # for checkboxes, check only values with the same option
+            # get the widget_type from the put request
```

Comment:
this is from put or post (checkboxes are stored with post) right?

Reply:
You are right, sometimes they are updated with put, but only if the additional input changes.